docker-compose.hobby.yml

```diff
@@ -117,6 +118,21 @@ services:
             - redis
             - clickhouse
             - kafka
+            - object_storage
+
+    object_storage:
+        image: minio/minio
+        ports:
+            - '19000:19000'
+            - '19001:19001'
+        volumes:
```

Comment:
For hobby, I'd add a named volume as with zookeeper and reference this rather than `./object_storage`, to be in line with the other services.